packages/devtools/web/index.html

```diff
@@ -85,6 +85,20 @@
 
           Plotly.extendTraces(domName, data, traces);
         }
+
+        function selectionExtendTraces(domName, x0, x1, y0, y1, orgTraces) {
+            var data = {};
+            var xData = [];
+            var yData = [];
+            var traces = [];
+
+            _copyTrace(x0, y0, orgTraces[0], xData, yData, traces);
+            _copyTrace(x1, y1, orgTraces[1], xData, yData, traces);
+
+            data = { x: xData, y: yData};
+
```

Comment:
nit: remove extra space before x or after yData (not sure what the proper style should be here but it should at least be consistent)

Reply:
Done removed the extra space.